dawn/renderer/components/PeripheralList.js

```diff
@@ -18,13 +18,18 @@ cleanerNames[PeripheralTypes.ServoControl] = 'Servo Controllers';
 cleanerNames[PeripheralTypes.RFID] = 'RFID';
 cleanerNames[PeripheralTypes.YogiBear] = 'Yogi Bear';
 
+const filter = [PeripheralTypes.TeamFlag];
+
 const handleAccordion = (array) => {
   const peripheralGroups = {};
+
   array.forEach((p) => {
```

Comment:
You can extract the filter includes check with
`array.filter(p => !filter.has(p.device_type)).forEach(p => /* ... */)`
assuming you converted the filter to a Set.